src/roswire/ros2/ros2.py

```diff
@@ -23,16 +25,24 @@ class ROS2:
     services: ROS2ServiceManager = attr.ib(init=False)
     launch: ROS2LaunchManager = attr.ib(init=False)
     _state_probe: ROS2StateProbe = attr.ib(init=False)
+    __package_source_extractor: ROS2PackageSourceExtractor = \
```

Comment:
If I remember correctly, `__foo` causes issues with `attr.ib`? It might be worth just sticking to a single underscore here if this causes problems.

Reply:
Done